setup.py

```diff
@@ -11,18 +11,22 @@
 limitations under the License.
 '''
 
-from setuptools import setup
+from setuptools import setup, find_packages
 
 PACKAGE_NAME = "warriorframework"
-PACKAGE_VERSION = "4.1.0" 
+PACKAGE_VERSION = "4.1.0"
 
 setup(
     name=PACKAGE_NAME,
     version=PACKAGE_VERSION,
     author="warriorframework org",
-    url="https://github.com/warriorframework/warriorframework",
-    install_requires=["pexpect==4.2", "requests==2.21.0", "selenium==3.8.0",
-                      "lxml==3.3.3", "paramiko==2.4.2", "pysnmp==4.4.9",
+    scripts=['warrior/Warrior'],
+    packages=find_packages(),
+    package_data={'':['**/*', '*']},
+    include_package_data=True,
+    url="https://github.com/warriorframework/warriorframework_py3",
+    install_requires=["pexpect==4.2", "requests==2.21.0", "selenium==2.53.0",
```

Comment:
The version have changed during the commit. Need to restore original version below.
"selenium==3.8.0",
"pysnmp==4.4.9",